### ISSUE TITLE
Icons not rendered

### ISSUE THREAD

**Issue by igorakkerman** [satr/intellij-idea-plugin-connector-for-aws-lambda#11]
Hi,

Unfortunately, on my system, the left hand side icons are not renedered as in your screenshots:

<img width="477" alt="2018-07-21 21_58_29-connector for aws lambda - ageindays-alexa" src="https://user-images.githubusercontent.com/10802670/43039630-9321dedc-8d31-11e8-9384-c2476d60e117.png">

These are my IntelliJ version details:
```
IntelliJ IDEA 2018.2 (Ultimate Edition)
Build #IU-182.3684.40, built on July 17, 2018
Windows 10 10.0
```

Cheers,
Igor

**Comment by satr** [comment]
Hi
I reproduced this on the Idea 2018.2 RC. Sometimes I noticed RC-versions of JetBrains products have different styles and layouts. I'll check if this can be handled in the plugin.
Thank you, I'll keep you informed

**Comment by satr** [comment]
Fixed in the release [1.7.RC1](https://github.com/satr/intellij-idea-plugin-connector-for-aws-lambda/releases/tag/v1.7.RC1)
There is an artefact with shown button's background. I will look at this later.
